lib/l10n/app_de.arb

```diff
@@ -154,11 +154,35 @@
 
   "pageHelpTitle": "Hilfe",
   "pageHelpContactUs": "Kontaktiere uns",
-  "pageHelpWhatIsMotisQuestion": "Was kann ich mit der MOTIS Ride App machen?",
-  "pageHelpWhatIsMotisAnswer": "Mit der MOTIS Ride App kannst du einige nützliche Dinge machen.",
-  "pageHelpWhereIsAccessibilityQuestion": "Wo finde ich die Einstellungen für Barrierefreiheit?",
-  "pageHelpWhereIsAccessibilityAnswer": "Diese App unterstützt eine Vielzahl von Einstellungen zur Barrierefreiheit, die vom Betriebssystem angeboten werden. Geh zu den Einstellungen deines Betriebssystems, um Screen Reader oder größere Schrift zu aktivieren!",
-
+  "pageHelpEmailSubject": "Anfrage von User-ID [{id}]",
+  "pageHelpQuestion1": "Was ist MOTIS?",
+  "pageHelpAnswer1": "MOTIS umfasst viele Tools zur Unterstützung des intermodalen Routings. Das Ziel ist es, bei der Planung von Reisen zu helfen, die öffentliche Verkehrsmittel, Bike-Sharing, Ride-Sharing und weitere Transportmittel umfassen.",
+  "pageHelpQuestion2": "Wie viel kosten Fahrten? Wie kann ich bezahlen?",
+  "pageHelpAnswer2": "Wir haben den Zahlungsprozess so konzipiert, dass er sich vollständig auf Benutzervereinbarungen stützt. Sowohl die Kosten als auch die Zahlungsmethode einer Fahrt können persönlich oder in unserem In-App-Chat ausgehandelt werden. Als Ausgangspunkt siehst du für die meisten Fahrten einen Preisvorschlag.",
+  "pageHelpQuestion3": "Wir haben uns auf einen Preis geeinigt, aber jetzt zahlt mein Fahrer ihn nicht. Was kann ich machen?",
```

Comment:
gendern? sollten wir wahrscheinlich

Reply:
Whoops, I'm surprised I didn't notice this myself, apparently Google Translate doesn't deal well with this sort of thing... Typical late-stage capitalism! Anyway, now it's much more readable.